package.json

```diff
@@ -78,6 +78,7 @@
     "yargs": "^16.1.0"
   },
   "dependencies": {
+    "crypto": "^1.0.1",
```

Comment:
[crypto-js](https://www.npmjs.com/package/crypto-js) works front and backend would this be a better option here for the same purpose? or it doesn't matter in this case?

Reply:
It serves the same purpose as window.crpyo. Its just that it does not use window object of JS and uses this npm package for crypto